src/twisted/conch/ssh/channel.py

```diff
@@ -12,14 +12,14 @@
 
 from zope.interface import implementer
 
-from twisted.python import log
 from twisted.python.compat import nativeString, intToBytes
 from twisted.internet import interfaces
+from twisted.logger import Logger
 
 
 
 @implementer(interfaces.ITransport)
-class SSHChannel(log.Logger):
+class SSHChannel(object):
```

Comment:
Note: There is perhaps a Hyrum's Law argument to be made that this is incompatible in some way, but I'm not going to make it.

I believe that this is okay because any calling code that does `isinstance(o, log.Logger)` is doing so to test whether it can call `logPrefix()`. However, the primary caller of the `log.Logger.logPrefix()` method is `callWithLogger()`, which doesn't actually do that isinstance check. So as long as we leave the `logPrefix()` methods in place I don't think we are giving up anything we already had.